go.mod

```diff
@@ -169,6 +169,6 @@ replace github.com/gogo/protobuf => github.com/gogo/protobuf v1.3.2
 
 // If you change this please also update .github/pull_request_template.md and
```

Comment:
should we remove this line since it doesnt look like there is anything in the pr template file that needs to change

Reply:
So this commit was accidentally tagged on lol, will remove it in the rebase.